ansible/roles/tower_demo/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+---
+- name: install pip via yum
+  yum:
+   name: python-pip
+
+- name: Install Tower cli
+  pip:
+   name: 'ansible-tower-cli'
+   version: '3.1.8'
+
+- name: Configure the tower cli file
+  template:
+    src: "{{ ANSIBLE_REPO_PATH }}/files/tower_cli.j2"
+    dest: "~/.tower_cli.cfg"
+    mode: 0640
+
+- name: copy the license file
+  copy:
+   src: "{{ ANSIBLE_REPO_PATH }}/files/tower_license"
+   dest: "/tmp/tower_license"
+
+- name: copy the private key
+  copy:
+    src: "{{ ANSIBLE_REPO_PATH }}/workdir/{{ guid + 'key' }}"
+    dest: /tmp/id_rsa
+    mode: 0640
+  
+- name: generate ansible hosts file
+  template:
+    src: "{{ ANSIBLE_REPO_PATH }}/configs/{{ env_type }}/files/hosts_template.j2"
+    dest: "/etc/ansible/hosts"
+
+- name: Add the license of tower
+  command: |
+     tower-cli setting modify LICENSE '{"eula_accepted" : "true", "company_name": "Red Hat Internal", "contact_email": "ber@redhat.com", "contact_name": "Jon Bersuder",  "instance_count": 16, "license_date": 1514782800, "license_key": "2e2995c0e36fa1846c34db031138d44e15dcf5b04c414c462d159f2648f579aa", "license_type": "enterprise", "subscription_name": "Ansible Tower by Red Hat, Standard (16 Managed Nodes)",  "trial": false }' -u admin -p "{{ tower_admin_password }}"
```

Comment:
can you split this line for readability ? i think you can do multilines using `\`

Reply:
"license_key": "2e2995xxxxxxxxx59f2648f579aa"

Shouldn't this be a variable  ? what is this license key ?